webdriver/tests/bidi/browsing_context/set_viewport/__init__.py

```diff
@@ -0,0 +1,21 @@
+from webdriver.bidi.modules.script import ContextTarget
+
+async def layout_viewport_dimensions(bidi_session, context):
```

Comment:
In `webdriver/tests/bidi/input/perform_actions/__init__.py` we already have a method called `get_viewport_rect`. Maybe we could make that a fixture for all BiDi tests and call it `viewport_dimensions`?

Reply:
https://sourcegraph.com/github.com/web-platform-tests/wpt/-/blob/webdriver/tests/bidi/browsing_context/capture_screenshot/__init__.py already has one too.

The best course of action here would be to create a fixture (or a helper method put in a more central place) and reuse it everywhere

Reply:
Should I do the refactoring before this PR? capture_screenshot/__init__.py could be split into get_viewport_rect and `get_dpr` and those can be combined in the test.

Reply:
Does anyone know why `get_viewport_rect` falls back to the `document.documentElement`? 

```
    expression = """
        ({
          height: window.innerHeight || document.documentElement.clientHeight,
          width: window.innerWidth || document.documentElement.clientWidth,
        });
    """
```